presto-main/src/test/java/com/facebook/presto/block/ColumnarTestUtils.java

```diff
@@ -13,33 +13,24 @@
  */
 package com.facebook.presto.block;
 
-import com.facebook.presto.metadata.FunctionManager;
+import com.facebook.presto.metadata.Metadata;
 import com.facebook.presto.spi.block.Block;
-import com.facebook.presto.spi.block.BlockEncodingSerde;
 import com.facebook.presto.spi.block.DictionaryBlock;
 import com.facebook.presto.spi.block.RunLengthEncodedBlock;
-import com.facebook.presto.spi.type.TypeManager;
-import com.facebook.presto.sql.analyzer.FeaturesConfig;
-import com.facebook.presto.type.TypeRegistry;
 import io.airlift.slice.DynamicSliceOutput;
 import io.airlift.slice.Slice;
 
 import java.lang.reflect.Array;
 import java.util.Arrays;
 
+import static com.facebook.presto.metadata.MetadataManager.createTestMetadataManager;
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
 
-final class ColumnarTestUtils
+public final class ColumnarTestUtils
```

Comment:
The change here is not shown in https://github.com/prestosql/presto/pull/901/commits/e05502dccd7c917fb7db45ae1793d0d088674938 . Is that done in other commits in PrestoSQL repo? Can you extract that commit as well? :)

Reply:
Removed the other changes coming in from that source file and retained just the single line change as in the commit